/**:
  ros__parameters:
    file_path: /home/siyuchen/Downloads/data/dataset/rivercloud_dataset/20240213/20240213_122305/IMUData_20240213_122305.log
    ros2bag_name: 20240213_122305_Blausteinsee_P4
    # translation from IMU body frame to the first antenna frame which is specified
    # in the configuration file of novatel
    # e.g. SETINSTRANSLATION ANT1 -0.057 0.531 -0.229 0.001 0.001 0.001 IMUBODY
    # dx = -0.057, dy = 0.531, dz = -0.229, unit is meter
    translation_imu_antenna1:
      - -0.057
      - 0.531
      - -0.229
    # rotation from IMU body frame to the output frame, i.e. vehicle frame.
    # Angles are given in degree. The order of rotations is Z, X, Y. All rotations are right-handed.
    # see https://docs.novatel.com/OEM7/Content/PDFs/OEM7_Commands_Logs_Manual.pdf page 1036 section 4.22
    # Rotation is extrinsic rotation, with fixed coordinate system.
    # e.g. SETINSROTATION RBV 180 0 -90
    # ax = 180, ay = 0, az = -90, unit is degree
    # but the correct rotation order is ZXY, so the values should be listed in this order
    rotation_imu_vehicle:
      - -90.
      - 180.
      - 0.
